fhirResourcesToLoad/o2teddy_careplan_hhs.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
CarePlan.text and CarePlan.category are required by USCore profiles

Reply:
Got it.